src/google/protobuf/compiler/php/php_generator.cc

```diff
@@ -297,6 +302,34 @@ std::string TypeName(const FieldDescriptor* field) {
   }
 }
 
+std::string PhpTypeName(const FieldDescriptor* field, bool is_descriptor) {
+  if (field->is_repeated()) {
+    return "array|\\Google\\Protobuf\\Internal\\RepeatedField";
```

Comment:
Currently, RepeatedField is for internal use. I'd like to only mention array as parameter for now.